Source/Core/InputCommon/ControllerEmu/StickGate.cpp

```diff
@@ -275,6 +281,41 @@ ReshapableInput::ReshapeData ReshapableInput::Reshape(ControlState x, ControlSta
   x -= m_center.x;
   y -= m_center.y;
 
+  // Let's just convert everything to positive then convert back afterwards
```

Comment:
This code, likewise, should probably go into its own function, that returns a pair, which can then be called like so:

```cpp
std::tie(x, y) = YourFunction();
```